.circleci/generate_circleci_config.py

```diff
@@ -21,6 +21,7 @@
     ("2.1.0-3", ["buster"]),
     ("2.1.1-2", ["buster"]),
     ("2.1.3-1", ["buster"]),
+    ("2.2.0-1", ["buster"]),
```

Comment:
```suggestion
    ("2.2.0-1.dev", ["buster"]),
```

Similar to https://github.com/astronomer/ap-airflow/commit/7d1c32cba40f21319e8e7e940ce65319808629cd#diff-801a26920fa2a3670f73ced4973ecb336bb2d11e8afd234edce7d8437a5ec752

Having `.dev` here will add more configs to `.circlecli/config.yml` to have nightly builds. You will have to run this file again after this change